types/metadataV12.go

```diff
@@ -83,6 +83,19 @@ func (m *MetadataV12) FindStorageEntryMetadata(module string, fn string) (Storag
 	return nil, fmt.Errorf("module %v not found in metadata", module)
 }
 
+func (m *MetadataV12) FindConstantValue(module Text, constant Text) ([]byte, error) {
```

Comment:
This looks like the repeat code. Why can't we reuse the one from Metadata V10

Reply:
done